### ISSUE TITLE
Get official

### ISSUE THREAD

**Issue by ForNeVeR** [ForNeVeR/wpf-math#4]
I'd like the people from [the outdated launchpad page](https://launchpad.net/wpf-math) to be aware about this resurrection of the original project. For that, I'd like the project description on Launchpad to include a link to GitHub.

To change the Launchpad description, I'd like to cooperate with the original project developer, @alexreg.

@alexreg, Alexander, hello and huge big **thank you** for your work on the original project! Could you please edit the project description on the Launchpad page?

I'm sort of library archaeologist, and I'm going to maintain the library fork, making it available on the new platforms and generally patch it the best I can.

**Comment by alexreg** [comment]
@ForNeVeR @gsomix Hey! Many thanks to both of you for your excellent work reviving this project.

Moving the project over to a more popular VCS and hosting site is an important first step forward, as well as applying all the pending bug fixes of course. I hope that the development directory I had sitting around on my old machine (which looks a bit more recent than the version on Launchpad) can be of help.

I've also seen some of the issues you've logged already, and it looks like you're taking the project forward in interesting new directions. Indeed, if it ends up being a general rendering library for mathematics in .NET, with sub-libraries for different renderers, then all the better!

I'm relatively busy these days (I have other FOSS projects), but if I can be of help with this project still, just let me know, and I'll try to contribute where and how I can. :)

**Comment by ForNeVeR** [comment]
Thank you, Alexander. I've added you as a contributor to the project, so feel free to make the changes you think are necessary.

Small note: I'd prefer anyone (including myself, you, and any other contributors) to not make direct commits into a **master** branch. We've adopted pull request strategy, so each PR will be reviewed by other contributors. I believe that's the standard FOSS project practice nowadays.

In the meantime, I'll check the changes from the bzr archive you've sent me and reintegrate the changes from there. Thanks for help!

The issue is now closed, because the Launchpad project page got updated.
